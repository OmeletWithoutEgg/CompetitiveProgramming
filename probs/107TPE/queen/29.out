-419689.01 2763366.79
