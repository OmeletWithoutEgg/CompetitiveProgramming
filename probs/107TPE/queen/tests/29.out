563058.05 27721833.70
